py-utils/src/utils/msg_bus/message_bus/message/__init__.py

```diff
@@ -0,0 +1,5 @@
+from __future__ import absolute_import

```

Comment:
All the code in this folder can be merged into message_bus/message.py

Reply:
merged